tests/devices/test_preprocess.py

```diff
@@ -254,7 +254,7 @@ def test_invalid_tensor_observable(self):
         with pytest.raises(qml.DeviceError, match="not supported on device"):
             validate_observables(tape, lambda obj: obj.name == "PauliX")
 
-    @pytest.mark.usefixtures("use_legacy_opmath")
+    @pytest.mark.usefixtures("legacy_opmath_only")
     def test_invalid_tensor_observable_legacy(self):
```

Comment:
This test case is an exact duplicate of the one above, except that it was marked `use_legacy_opmath`, so we can probably delete this one entirely.

Reply:
We should be able to do the same for all the duplicate tests where the "new" one is decorated with the `"use_new_opmath"` fixture. I'll start looking around.